_posts/2025-06-04-cockpit-340.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cockpit 340
+author: Venefilyn
+date: "2025-06-04"
+tags: cockpit, podman, files, machines, ostree
+slug: cockpit-340
+category: release
+summary: "Handle unsupported storage partitions, link quadlet logs to their service, source mode for virtual networks, various UX improvements."
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 340, cockpit-podman 107, cockpit-machines 333, and cockpit-ostree 211:
+
+## Storage: Prevent modifying partitions in unsupported places
+
+It was previously confusing and somewhat dangerous when storage page listed LVM2 Logical Volumes as having free space. This is now changed to include a notice and not display these partitions to prevent mishaps when we cannot support it.
+
+![Screenshot showing off a notice inside a GPT partitions card saying "Unexpected partitions" in bold, followed by a description saying "Partitions are not supported on this block device. If it is used as a disk for a virtual machine, the partitions must be managed by the operating system inside the virtual machine."]({{ site.baseurl }}/images/340-storage-unsupported-partitions.png)
+
+## UX improvements
+
+- We now warn the user when renaming volume groups that contain the root filesystem as it might break the next boot.
+- Health card items in overview page is now aligned correctly.
```

Comment:
These are completely unrelated. The second one is just a minor bug fix, and should IMHO be dropped. The remaining one could be "Storage: Rename safety warning".